core/vm/contracts_suave_eth.go

```diff
@@ -355,6 +355,13 @@ func executableDataToCapellaExecutionPayload(data *engine.ExecutableData) (*spec
 	}, nil
 }
 
+type SubmissionResults struct {
+	StatusCode    uint64
+	Destination   string
+	RoundTripTime *big.Int
```

Comment:
instead of bigInt complexity, perhaps go for uint64 in millis? or alternatively record `startTimeUnix` and `endTimeUnix` 🤔

Reply:
totally great suggestions! just threw together something quick to illustrate